cluster-autoscaler/SYNC-CHANGES/SYNC_CHANGES-1.26.1.md

```diff
@@ -0,0 +1,24 @@
+<!--- For help refer to https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md?plain=1 as example --->
+
+- [v1.26.1](#v1261)
```

Comment:
pls add this to the SYNC_CHANGES-1.26.md only.
No need to create a new file

Reply:
done